src/widgets/Dialogs.js

```diff
@@ -180,7 +180,7 @@ define(function (require, exports, module) {
             }
         } else if (brackets.platform === "mac") {
             // CMD+D Don't Save
```

Comment:
Hello Marcel,

according to Apple CMD-Backspace is the default for Don't Save. Or did I got it wrong :/

Anyhow I have updated the comment. Thanks for the tip.


Regards,
Lucas L.

> On 01 Feb 2015, at 14:29, Marcel Gerber <notifications@github.com <mailto:notifications@github.com>> wrote:
> 
> In src/widgets/Dialogs.js <https://github.com/adobe/brackets/pull/10459#discussion_r23897530>:
> 
> > @@ -180,7 +180,7 @@ define(function (require, exports, module) {
> >              }
> >          } else if (brackets.platform === "mac") {
> >              // CMD+D Don't Save
> You should rather call this CMD+Delete, right?
> Also, I think CMD+D still should trigger Don't Save, shouldn't it?
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/adobe/brackets/pull/10459/files#r23897530>.
>